.env

```diff
@@ -1 +1,4 @@
-PORT=3000
```

Comment:
This file shouldn't be pushed to GitHub (add it to .gitignore)

It usually contains sensitive information.

Reply:
I know, but according to the `.gitignore` file in the repository we should leave it in for learning purposes.